src/interpreter.rs

```diff
@@ -64,18 +64,6 @@ pub enum Error {
     #[error("verify operation failed")]
     Verify,
 
-    #[error("equal verify operation failed")]
```

Comment:
Is the distinction between these really not relevant? I could see it providing some useful information to someone in the process of iteratively developing a script.

Reply:
The commit before this pairs each `interpreter::Error` with the opcode that triggered it, so the distinction remains.

At the level of interpreting a single opcode, you know which opcode triggered the error, so you don’t need the distinction there.